### PR TITLE
feat(PaymentCard): new card designs & new products

### PR DESCRIPTION
**Pull request by langz**
This PR implements the new card designs, see https://www.figma.com/file/461cAN5Qc3Nks4ztZ9pjtM/Eufemia---Payment-Cards?node-id=4-503&t=SmsUdZPPfptWSCoT-0 for the sketches.

A quick history lesson:
The previous design seen in v9, https://eufemia.dnb.no/uilib/extensions/payment-card/demos/, had a more "generic approach" to card designs. It didn't exactly look like the "real products", in terms of colors, composition, etc. This was done on purpose, as the intention was that if done like this(not 100% matching any real products) then we wouldn't have to update the payment cards every time new card products and designs came around.

However this is not how they want it anymore, so as part of this PR, the payment cards should look more like the actual cards seen in https://bc.dnb.no/brandcenter/no/dnbvisualguidelines/component/default/2882 

It can be demoed [here](https://eufemia-git-feat-update-payment-card-designs-eufemia.vercel.app/uilib/extensions/payment-card/demos/)(just scroll down on the page, and look for or search for the productIDs from the [design](https://www.figma.com/file/461cAN5Qc3Nks4ztZ9pjtM/Eufemia---Payment-Cards?node-id=0-1&t=VQ2i1nzuUzE2y2Cb-0), like `VX4`, `BK1`, `VB1`, `VB5`, etc.

### CODE REVIEW

**Comment by langz** [comment]
What do we think of this implementation?
@tujoworker @joakbjerk 
I'm starting to get quite eager to release v10.
If it turns out there's a few minor "issues" in this PR, like productcode and minor design flaws, then I would rather want to fix it in a upcoming bugfix release, instead of delay release v10.

**Review by joakbjerk — APPROVED** [review]

**Comment by joakbjerk** [comment]
> What do we think of this implementation? @tujoworker @joakbjerk
> 
> I'm starting to get quite eager to release v10. If it turns out there's a few minor "issues" in this PR, like productcode and minor design flaws, then I would rather want to fix it in a upcoming bugfix release, instead of delay release v10.

Looks good very good 🔥 🙏

080 - Private Banking Mastercard seems to be missing the Bankaxept logo
Screenshot 2023-05-03 at 10 35 30

And just a dumb question regarding the Mastercard Gold 😅 The corporate text is missing next to the DnB logo, but I also see that the card is marked as deprecated in Confluence, so this might not be an issue at all?
Screenshot 2023-05-03 at 10 36 18

This is not something we need to fix now as you said, could easily come in an later release, so we can prioritize releasing V10. Just wanted to note this down somewhere 🤔

**Comment by tujoworker** [comment]
I find this base64 convert nice to use: https://base64.guru/converter/encode/image/jpg

**Comment by langz** [comment]
> I find this base64 convert nice to use: https://base64.guru/converter/encode/image/jpg

Good good, I just ran all the images through https://compressjpeg.com/ and then the https://base64.guru/converter/encode/image/jpg.

Result is:
- my-first.ts: 7 966 bytes
- ung.ts: 9 807 bytes
- youth.ts: 3 631 bytes

**Review by tujoworker — APPROVED** [review]
Impressive work @langz 👌

File: packages/dnb-eufemia/src/extensions/payment-card/icons/Bedrift.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
...2.965 19.782V21H24.421V11.2H22.783ZM20.767 19.81C19.633 19.81 18.877 18.872 18.877 17.5C18.877 16.128 19.633 15.19 20.767 15.19C21.957 15.19 22.783 16.142 22.783 17.5C22.783 18.858 21.957 19.81 20.767 19.81ZM29.5916 13.874C28.6816 13.874 27.9116 14.364 27.5476 15.134V14H26.1056V21H27.7436V17.066C27.7436 15.876 28.4296 15.288 29.5496 15.288C29.8156 15.288 30.0536 15.316 30.2356 15.358V13.944C30.1236 13.916 29.8856 13.874 29.5916 13.874ZM32.3216 13.3C32.8956 13.3 33.3856 12.824 33.3856 12.25C33.3856 11.676 32.8956 11.186 32.3216 11.186C31.7476 11.186 31.2716 11.676 31.2716 12.25C31.2716 12.824 31.7476 13.3 32.3216 13.3ZM31.5096 21H33.1476V14H31.5096V21ZM38.1287 12.39C38.3807 12.39 38.6887 12.432 38.8707 12.502V11.214C38.6467 11.144 38.1427 11.074 37.7647 11.074C36.3087 11.074 35.5107 11.858 35.5107 13.3V14H34.4887V15.316H35.5107V21H37.1487V15.316H38.7867V14H37.1487V13.412C37.1487 12.754 37.4847 12.39 38.1287 12.39ZM43.3546 19.81C42.6266 19.81 42.2766 19.488 42.2766 18.802V15.316H44.0686V14H42.2766V11.816H40.6386V14H39.5046V15.316H40.6386V18.858C40.6386 20.314 41.5066 21.126 43.0606 21.126C43.5506 21.126 43.9286 21.084 44.1946 20.972V19.684C43.9706 19.768 43.6766 19.81 43.3546 19.81Z"
```
[the remark this answers is not in the document]
Comment (langz) [reply packages/dnb-eufemia/src/extensions/payment-card/icons/Bedrift.js]: Done ✅

File: packages/dnb-eufemia/src/extensions/payment-card/icons/Intro.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from 'react'
```
[the remark this answers is not in the document]
Comment (langz) [reply packages/dnb-eufemia/src/extensions/payment-card/icons/Intro.js]: I've added a commit for this now, https://github.com/dnbexperience/eufemia/pull/2264/commits/e26b21bf114324fd93bb44f97f6ee3d12c8598af

**Pull request merged**